src/utils/ProtectedRoute.tsx

```diff
@@ -0,0 +1,19 @@
+
+import React from 'react'
+
+import { Navigate } from 'react-router-dom'
+
+import LoginService from '../services/LoginService'
+
+
+const ProtectedRoute = ({ children }: {children: JSX.Element} ) => {
+
+  !LoginService.isLoggedIn()
+  if(!LoginService.isLoggedIn()) {
```

Comment:
You can write this in a shorter way, please use the ternary operator in this case.

Reply:
Ok, I will fix this.